lib/IRGen/IRGenDebugInfo.cpp

```diff
@@ -1062,6 +1062,20 @@ class IRGenDebugInfoImpl : public IRGenDebugInfo {
         llvm::dwarf::DW_LANG_Swift, nullptr, MangledName);
   }
 
+  llvm::DIType *createFixedValueBufferStruct(llvm::DIType *PointeeTy) {
+    unsigned Line = 0;
+    unsigned PtrSize = CI.getTargetInfo().getPointerWidth(0);
+    llvm::DINode::DIFlags Flags = llvm::DINode::FlagArtificial;
+    llvm::DIFile *File = MainFile;
+    llvm::DIScope *Scope = TheCU;
+    llvm::Metadata *Elements[] = {DBuilder.createMemberType(
+        Scope, "contents", File, 0, PtrSize, 0, 0, Flags, PointeeTy)};
+    return DBuilder.createStructType(
+        Scope, "$swift.fixedbuffer", File, Line, 24 * 8, 0, Flags,
```

Comment:
I think `24 * 8` is wrong on 32-bit -- it should be `3 * WordSize * 8` bits in size no?